hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java

```diff
@@ -8682,15 +8706,15 @@ protected void decrementFlushesQueuedCount() {
    * {{@link #enableInterrupts()}.
    */
   void disableInterrupts() {
-    regionLockHolders.computeIfPresent(Thread.currentThread(), (t, b) -> false);
+    intrRegionLockHolders.computeIfPresent(Thread.currentThread(), (t, b) -> false);
```

Comment:
I thought earlier version of having single map was simpler. It seems bit confusing now that we have two maps `intrRegionLockHolders` for interruptible threads and `nonintrRegionLockHolders` for non-interruptible threads. 

However, here in disableInterrupts(), we update interruptible as non-interruptible so even though we have `intrRegionLockHolders` for interruptible threads, we can make them non-interruptible and still have them stay in `intrRegionLockHolders` map 🤔

In a way, the interruptible info is only meaningfully exercised by `interruptRegionOperations()` for which the previous version of the PR was good enough I think.